### DISCUSSION TITLE
Account restricted for violation of terms of service

### DISCUSSION THREAD

**Discussion by tawfiqkhalilieh** [codewars/codewars.com#3039] [Q&A]
My account was restricted yesterday. can you please tell me why and what can I do in this case ?

**Comment by Blind4Basics** [comment]
> My account was restricted yesterday. can you please tell me why and what can I do in this case ?

which one of your many accounts are you talking about?

**Reply by tawfiqkhalilieh** [reply]
sorry I forgot to mention it 😅.

here is my account: https://www.codewars.com/users/tawfiq2

**Reply by Blind4Basics** [reply]
* this one got locked just 50min ago, not "yesterday" (unless you're in a weird timezone...)
* it looks like you didn't get the irony in my question

**Reply by tawfiqkhalilieh** [reply]
hmm, is there anything I can do to unrestrict it ?

**Reply by Blind4Basics** [reply]
considering the solutions used in there, no. And I suggest you stop creating alter accounts, as well as posting solutions you found on the internet (or on your previous accounts. Which mostly boils down to the same anyway).

Cheers

**Reply by Blind4Basics** [reply]
btw, you should delete all the previous accounts. Not just change their names...

**Reply by tawfiqkhalilieh** [reply]
According to the TOS it's legal to share my solutions so my main account should not be restricted

**Reply by Blind4Basics** [reply]
are you _really_ serious...? :rofl: 
I'd now suggest you stop searching excuses/holes in the TOS and start solving problems on your own. ;)

**Reply by tawfiqkhalilieh** [reply]
Even though I violated some terms of service when I created the new accounts, your response is unprofessional.

**Reply by Blind4Basics** [reply]
Sorry about that. 'Good thing I'm not paid for this, I guess. ;)

**Reply by tawfiqkhalilieh** [reply]
So it might be better to not talk by the name of Codewars if you won't talk professionally.

**Reply by Blind4Basics** [reply]
Well, things are complicated around here. I'm not paid for this, but I'm technically the one who's handling copycats/cheaters. Hence, you see me answering here when the question is about restricted accounts.

Now, in case it wasn't clear enough:
* alter accounts are "legit" but strongly discouraged. Because in 9 cases out of 10, they are used in dubious ways. Just like you're (still) currently doing.
* on the other hand, and as I pointed above, stealing solutions over the internet is _not_ allowed. 
* your original account _is_ the account of a copycat
* that makes any of your other accounts also eligible for restrictions. Especially considering how you "fill" them.

Now, why would you ever need those 10+ accounts of yours? In a legit way, I mean.
